interview-questions.md

```diff
@@ -8,47 +8,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is git? What is the difference between git and Github?
 
   Your answer:
+Git is a way to save code that is being worked on by one or more parties. GitHub is the online cloud where the information can be accessed, stored, and saved.
 
   Researched answer:
-
-
+ Global Information Tracker, also known as Git, is what is called a version control system. Version control systems are important for programmers because they are able to track and log changes made on files and they also give one the ability to restore or review earlier versions of said file. Git is a vcs that is located and operates on your computer.  
+  GitHub is different because it's an online Git repository hosting service. It allows users to make and store Git repositories online so that others may access them to work on, edit, etc.
 
 2. What types of evaluations will return a Boolean value?
 
   Your answer:
+    Types of evaluations that will return Boolean values are things like is 6 > 8 or does 7 === 7 and things like that. Basically anything that returns a true or false statement, returns a Boolean value.
```

Comment:
Yup for sure. And these things are specifically called relational, logical, and equality operators.